zerver/migrations/0563_zulipinternal_can_delete.py

```diff
@@ -0,0 +1,13 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0562_remove_realm_create_web_public_stream_policy"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            "UPDATE zerver_realm SET delete_own_message_policy = 1 where string_id = 'zulipinternal'"
```

Comment:
If we're doing this in a migration, we should presumably modify `zerver/lib/server_initialization.py` as well to change it for new realms?